src/llvm-late-gc-lowering.cpp

```diff
@@ -2386,8 +2386,7 @@ bool LateLowerGCFrame::CleanupIR(Function &F, State *S, bool *CFGModified) {
                     if (isLoadFromConstGV(LI, task_local) && getLoadValueAlign(LI) < 16) {
                         Type *T_int64 = Type::getInt64Ty(LI->getContext());
                         auto op = ConstantAsMetadata::get(ConstantInt::get(T_int64, 16));
-                        LI->setMetadata(LLVMContext::MD_align,
-                                        MDNode::get(LI->getContext(), { op }));
+                        LI->setMetadata(LLVMContext::MD_align, MDNode::get(LI->getContext(), { op }));
```

Comment:
This load is also worth assessing, if we're emitting constant globals with less alignment than 16.

Reply:
That is the alignment of the loaded pointer (which in this case is a `jl_datatype_t*`), not the GlobalValue. So we know this is true, since 16 is the expected heap alignment of those pointers.